plugins/ca-changesets/plugin.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "ca-changesets",
+  "version": "0.0.1",
+  "description": "Culture Amp plugin providing the binary included in the npm package @changesets/cli",
+  "readme": "README.md",
+  "packages": ["nodejs"],
+  "create_files": {
+    "{{.Virtenv}}/bin/changeset": "bin/ca-changeset-wrapper"
+  },
+  "env": {
+    "DEVBOX_COREPACK_ENABLED": "true",
+    "PATH": "{{ .Virtenv }}/bin:$PATH"
+  }
```

Comment:
You need to expose the script so it can be used in repos with the plugin. I think the way to do it is with a `shell#scripts` value like we've done [in the ca-kafka-local plugin](https://github.com/cultureamp/devbox-extras/blob/main/plugins/ca-kafka-local/plugin.json#L36-L38)

Reply:
If you add a script called `changeset`

Reply:
The script is on the path due to where I place it in virtenv, and then the PATH manipulation.

This is so that any commands/args can be provided in a manner that matches the changeset documentation, if we add it as a devbox script the invocations would require a double dash before commands/args e.g. `devbox run changeset -- init` as opposed to how I have built it `changeset init`

Reply:
oh nice I missed that, sounds good